playbooks/eucalyptus_basic_server.yml

```diff
@@ -0,0 +1,77 @@
+---
+# for single server community
+# This file based on `edx_sandbox.yml`
+
+- name: Configure instance(s)
+  hosts: all
+  become: True
```

Comment:
@mrgnr is [here](https://github.com/appsembler/configuration/blob/appsembler/eucalyptus/develop/playbooks/roles/sudo/tasks/main.yml) is a separate role in `configuration`. I think if we include it, we can avoid to have to edit the `sudoers` file on every new deploy.